samples/kotlin_compose/build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "io.customer.android.sample.kotlin_compose"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "io.customer.android.sample.kotlin_compose"
+        minSdk = 24
```

Comment:
Are we able to set the min to the min version of our SDK? I think this will improve the quality of our testing with these apps being able to test the min SDK version of our SDK is appropriate.

Reply:
I think we can use it, but I think that would be more of a restriction rather than flexibility. Because we would want our SDK to be compiled against different SDK versions, if all our sample compiles against one then we lose the change to test it.

Reply:
I do think it's important that at least 1 of our sample apps uses the min SDK version. 

Maybe we make the Java sample app do that? We designate our Kotlin sample app to be more modern and use modern tools and SDK versions?

Reply:
Java app already uses [21 for `minSdk`](https://github.com/customerio/customerio-android/blob/main/samples/java_layout/build.gradle#L19) which is same as our [`minSdk` in SDK](https://github.com/customerio/customerio-android/blob/main/buildSrc/src/main/kotlin/io.customer/android/Configurations.kt#L6).

Are you trying to say we should reference `Configurations.minSdk` from Java sample app as well?

Reply:
:facepalm: I apologize, I made a mistake that caused this confusion. 

For some reason, I was thinking the min sdk of our sdk was lower then 21. After checking it again, it is indeed 21 which matches the Java sample app as expected. 

Sorry for causing confusion here. We are good with both apps 👍